### PR TITLE
Cover 45022

### PR DESCRIPTION
**Pull request by huiran0826**
OCP-45022 tests the multicast pods function across the nodes. 
Add some steps to OCP-12926 and cover this function.

Meanwhile, revert the codes in https://github.com/openshift/verification-tests/pull/2901, reasons please see https://github.com/openshift/verification-tests/pull/2901#issuecomment-1160083865

Testlogs:
OCP-12926 -- https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4487/console
OCP-12929 -- https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4488/console

@weliang1 @jechen0648  Could you take a look? Thanks a lot!

### CODE REVIEW

**Comment by huiran0826** [comment]
@weliang1 @jechen0648 Could you take a look? Thanks!

**Comment by huiran0826** [comment]
@jechen0648 Actually not, after this PR merge, I would add one more step in Polarion case OCP-12926 for this change and then inactive case OCP-45022.

File: features/networking/multicast.feature [hunk 1]
```diff
@@ -13,18 +13,28 @@ Feature: testing multicast scenarios
...
   Scenario: OCP-12926 pods should be able to subscribe send and receive multicast traffic
     # create some multicast testing pods
+    Given I store the schedulable workers in the :nodes clipboard
     Given I have a project
     And evaluation of `project.name` is stored in the :proj1 clipboard
     Given I obtain test data file "networking/multicast-rc.json"
-    When I run the :create client command with:
-      | f | multicast-rc.json |
+    When I run oc create over "multicast-rc.json" replacing paths:
+      | ["spec"]["replicas"] | 2                       |
```
Comment (weliang1) [inline features/networking/multicast.feature:21]: @huiran0826 If this step only create two pods  which are in the different nodes, do we still need a case for the pods are in the same node? Other case may covered. To make things simple, keep OCP-12926 as test pods in the same node, and using OCP-45022 to test Multicast traffic cross nodes.  WDYT?

Comment (huiran0826) [reply features/networking/multicast.feature:21]: @weliang1  Actually  OCP-12926 will cover both situation, same nodes and different nodes. 
Currently we only have this OCP-45022 for multicast as manual left, now I prefer to just update this case than to create a new case in go which might need a lot of effort and research as we don't have the basic utilizes for this kind of cases.

**Comment by huiran0826** [comment]
@zhaozhanqi @anuragthehatter Would you please take a look?  Thanks!

File: features/networking/multicast.feature (hunk 1 above)
Comment (weliang1) [reply features/networking/multicast.feature:21]: OK. 
/lgtm

**Pull request merged**